DailyReportWeb/Helpers/ReportRunner.cs

```diff
@@ -30,5 +32,40 @@ public static void RunReport(string id)
                 task.Run();
             }
         }
+
+        public static bool IndividualReportConfirmedByAll(Policy policy)
+        {
+            var draftsInfo = policy.GeneratedProperties.IndividualDrafts;
+            if (draftsInfo.Exists(i => i.Confirmed == false))
+                return false;
+
+            return true;
+        }
+
+        public static bool SetIndividualDraftConfirmation(string key, Policy policy, string policyPath)
+        {
+            try
+            {
+                var draftsInfo = policy.GeneratedProperties.IndividualDrafts;
+                var draft = draftsInfo.Find(d => d.UserKey == key);
+                draft.Confirmed = true;
+                policy.SaveToFile(policyPath);
+
+                return true;
+            }
+            catch (Exception)
+            {
+                return false;
+            }
+        }
+
+        public static bool CheckConfirmed(string key, Policy policy)
+        {
+            {
+                var draft = policy.GeneratedProperties.IndividualDrafts.Find(dr => dr.UserKey == key);
+
+                return draft.Confirmed;
+            }
+        }
```

Comment:
Metodele astea 3 din acest fisier pot fi puse ca si extension methods pe policy. 

Pentru asta ai nevoie de o clasa statica cu numele PolicyExtensions (poti sa o denumiesti si altfel, dar e mai intuitiv asa). Metodele trebuie sa fie si ele statice, sa aiba Policy ca prim parametru si parametrul sa aiba un `this` in fata: `this Policy policy`